package.json

```diff
@@ -29,7 +29,9 @@
   },
   "dependencies": {
     "dotenv": "^16.0.1",
-    "mixpanel-browser": "^2.43.0"
+    "mixpanel-browser": "^2.43.0",
+    "webpack": "^5.74.0",
+    "webpack-5": "^0.0.0"
```

Comment:
Is it required to include both the dependencies?

Reply:
No, I just checked and it working fine with only one so I removed the other one.